src/gan/querygan_pyt.py

```diff
@@ -0,0 +1,367 @@
+#!usr/bin/env/python

```

Comment:
Ultimately, what's the difference betwee n `querygan.py` and `querygan_pyt.py`? are both needed? can one be deleted?

Reply:
Moving from querygan.py to the _pyt version as we speak. The non-pyt one used Theano for the generator system. I've got both right now because that move is still in transition.